onebusaway-android/src/main/res/layout/floating_bike_info_window.xml

```diff
@@ -21,8 +21,7 @@
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textAppearance="@style/TextAppearance.Design.Tab"
-        android:text="Floating bike"/>
+        android:text="@string/floating_bike_title"/>
```

Comment:
Let's tweak this formatting a bit more, both for floating bikes and stations:

- [ ] Let's swap the "Rent this bike" and the bike ID locations, so the ID is the title  This will match the vehicle marker balloon better (see below - in fact, just use those exact styles).  Additionally, you can add the chevron (arrow) icon in the bottom right corner to indicate that the balloon is clickable - the end result should look something like the below (left is current, right is new):

![image](https://user-images.githubusercontent.com/928045/29041411-be10b05a-7b80-11e7-90e3-dbe81c5a5e07.png)

![image](https://user-images.githubusercontent.com/928045/29042643-c400d7f6-7b85-11e7-81fa-69769bd370db.png)


- [ ] For stations, let's make it look something like the below (left is current, right is new):

![image](https://user-images.githubusercontent.com/928045/29042659-d7704a2e-7b85-11e7-8f8f-3b180309fcd3.png)


I'm open to alternate design/layouts too if anyone has any better suggestions.